packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1409,14 +1413,14 @@
   </labels>
   <labels>
     <fullName>quantic_GeneratedAnswerOn</fullName>
-    <value>Generated answer ON</value>
+    <value>ON</value>
```

Comment:
We can't change the values just because they'd fit better xD

Also please don't run prettier on this file it looks stupid to have text on multiple lines.